### DISCUSSION TITLE
Budibase Cloud - database not found 404 error

### DISCUSSION THREAD

**Discussion by stellarfinn** [Budibase/budibase#4541] [Help]
I found topics covering this in the previous discussions but they were closed.

I have an app and database set to public and I cannot view it unless logged in to Budibase.

App at:
HTTPS://ciaoimports.budibase.app/app/ciaob2b

Also, I setup an app template and set the screens and database to public and this app also gives the same 404 error.

Any help much appreciated!

**Comment by trashidev** [comment]
I'm having this issue too. I stayed logged off on my phone to test that it works properly in public mode. It was working perfectly, but as of a few days ago its not. I am getting "Database does not exist", "404" when bringing up my app logged off on my phone. Logging in on my phone, the app works as before. I have confirmed all routes and databases are set to "public". 

I'm using Budibase cloud as well. 

I really don't want to have to make everyone to make an account and log in, just an access key is more than enough for my use...

**Comment by mjashanks** [comment]
@trashidev  @stellarfinn  - can confirm - we'll get this looked at asap.

**Reply by stellarfinn** [reply]
Thank you! I'm trying to convince my company's owner to migrate some of our manual tasks into a Budibase app and I need to show him the demo of the app I built to force his hand. Your help is much appreciated.

**Reply by trashidev** [reply]
> @trashidev @stellarfinn - can confirm - we'll get this looked at asap.

I just set up a self hosted version to test, and the issue only seems to affect cloud. Exporting the app and importing to the self-hosted version it works again.

**Reply by stellarfinn** [reply]
@trashidev Thanks for sharing. I was wondering if it was isolated to Budibase Cloud users.

**Reply by stellarfinn** [reply]
@mjashanks Any updates on this?

**Reply by mjashanks** [reply]
Hey @stellarfinn this was fixed yesterday - your link above works now!

**Reply by stellarfinn** [reply]
@mjashanks You rock! Thanks so much!
